compressor/parser/beautifulsoup.py

```diff
@@ -10,7 +10,9 @@ def __init__(self, content):
         try:
             from bs4 import BeautifulSoup
 
-            self.soup = BeautifulSoup(self.content, "html.parser")
+            # Disable multi_valued_attributes
+            # http://www.crummy.com/software/BeautifulSoup/bs4/doc/#multi-valued-attributes
+            self.soup = BeautifulSoup(self.content, "html.parser", multi_valued_attributes={})
```

Comment:
The docs linked seem to suggest using None here... 

> You can force all attributes to be stored as strings by passing multi_valued_attributes=None as a keyword argument into the BeautifulSoup constructor:
>
> ``` 
> no_list_soup = BeautifulSoup('<p class="body strikeout"></p>', 'html.parser', multi_valued_attributes=None)
> ```

Reply:
```suggestion
            self.soup = BeautifulSoup(self.content, "html.parser", multi_valued_attributes=None)
```

Yeah, let's do it the documented way.